modules/monitoring-creating-user-defined-alerting-rules.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * monitoring/managing-alerts.adoc
+
+:_content-type: PROCEDURE
+[id="creating-user-defined-alerting-rules_{context}"]
+= Creating user-defined alerting rules
+
+As a cluster administrator, you can create user-defined alerting rules based on platform metrics. 
```

Comment:
```suggestion
As a cluster administrator, you can create new alerting rules based on platform metrics. 
```

Reply:
I agree that this could be confusing. I've revised throughout to remove "user-defined" in these topics.